OpenRA.Game/Game.cs

```diff
@@ -679,21 +679,22 @@ static void RenderTick()
 				}
 
 				// worldRenderer is null during the initial install/download screen
-				if (worldRenderer != null)
+				// World rendering is disabled while the loading screen is displayed
+				// Use worldRenderer.World instead of OrderManager.World to avoid a rendering mismatch while processing orders
+				if (worldRenderer != null && !worldRenderer.World.IsLoadingGameSave)
```

Comment:
I know we can technically just subtract `render_widgets` from `render`, but for easier distinction, maybe add a new `using (new PerfSample("render_world"))`?